src/main/java/org/myrobotlab/framework/repo/Repo.java

```diff
@@ -141,6 +141,7 @@ protected Repo() {
       remotes.add(new RemoteRepo("bintray", "https://jcenter.bintray.com", "the big kahuna"));
       remotes.add(new RemoteRepo("bintray2", "https://dl.bintray.com", "more big kahuna"));
       remotes.add(new RemoteRepo("myrobotlab", "http://repo.myrobotlab.org/artifactory/myrobotlab", "all other mrl deps"));
+      remotes.add(new RemoteRepo("sarxos", "http://oss.sonatype.org/content/repositories/snapshots", "for sarxos webcam"));

```

Comment:
we won't need this additional repo if we use 0.3.12 of sarxos

Reply:
does "not" work with raspi